embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -426,6 +426,8 @@ public TaskReport run(TaskSource taskSource,
         LastRecordStore lastRecordStore = null;
 
         try (JdbcInputConnection con = newConnection(task)) {
+            con.before_load();
```

Comment:
`newConnection` is called by `AbstractJdbcInputPlugin#transaction` first, then called by `AbstractJdbcInputPlugin#run`.
So we should check timezone in the `transaction` method.
<pre>
    @Override
    public ConfigDiff transaction(ConfigSource config,
            InputPlugin.Control control)
    {
        ...
        Schema schema;
        try (JdbcInputConnection con = newConnection(task)) {
            // TODO incremental_columns is not set => get primary key
            schema = setupTask(con, task);
        } catch (SQLException ex) {
            throw Throwables.propagate(ex);
        }

        return buildNextConfigDiff(task, control.run(task.dump(), schema, 1));
    }
</pre>

Now `setupTask` method is `private`, but I think the method is appropriate to check timezone if it is changed to `protected`.